### PR TITLE
Create User

### PR DESCRIPTION
**Pull request by BlagovestDamyanov**
After the update, a user can be created. Data for CurrentHomeId and CurrentTownId have been added.The method  GetUserByEmail has also been implemented

### CODE REVIEW

File: MentorBook.Web/Controllers/UserController.cs [hunk 1]
```diff
@@ -93,7 +93,10 @@ public ActionResult InsertUser([FromBody] NewUserQM user)
             dbUser.LastName = user.LastName;
             dbUser.Email = user.Email;
             dbUser.Phone = user.Phone;
+            dbUser.CurrentTownId = (int)user.CurrentTownId;
+            dbUser.HomeTownId = (int)user.HomeTownId;
             dbUser.DateOfBirth = user.DateOfBirth;
+
```
Comment (MilenLambev) [inline MentorBook.Web/Controllers/UserController.cs]: Do we need that extra line?

File: MentorBook.Web/Controllers/UserController.cs [hunk 2]
```diff
@@ -93,7 +93,10 @@ public ActionResult InsertUser([FromBody] NewUserQM user)
             dbUser.LastName = user.LastName;
             dbUser.Email = user.Email;
             dbUser.Phone = user.Phone;
+            dbUser.CurrentTownId = (int)user.CurrentTownId;
```
Comment (MilenLambev) [inline MentorBook.Web/Controllers/UserController.cs]: Why do we have casting here, isn't user.CurrentTownId integer?

File: MentorBook.Business/UserService.cs [hunk 3]
```diff
@@ -39,20 +39,20 @@ public bool InsertUser(User user)
...
             {
-                // Add date created to the user
+                //Add date created to the user
                 user.DateCreated = DateTime.Now;
 
-                // Check if current town id and home town id exists in the database
+                //Check if current town id and home town id exists in the database
 
-                // Insert user to the database
+                 //Insert user to the database
```
Comment (MilenLambev) [inline MentorBook.Business/UserService.cs]: Do we need that space?

File: MentorBook.Web/Controllers/UserController.cs (hunk 2 above)
Comment (MilenLambev) [inline MentorBook.Web/Controllers/UserController.cs]: Why do we have casting here, isn't user.CurrentTownId integer?

File: MentorBook.Web/Controllers/UserController.cs (hunk 1 above)
Comment (MilenLambev) [inline MentorBook.Web/Controllers/UserController.cs]: Do we need that extra line?

**Review by BlagovestDamyanov — COMMENTED** [review]
solving a problem

**Review by BlagovestDamyanov — COMMENTED** [review]
solving a problem

**Pull request merged**